doc/source/user_guide/timeseries.rst

```diff
@@ -1780,6 +1777,50 @@ natural and functions similarly to :py:func:`itertools.groupby`:
 
 See :ref:`groupby.iterating-label` or :class:`Resampler.__iter__` for more.
 
+.. _timeseries.adjust-the-start-of-the-bins:
+
+Use `origin` or `offset` to adjust the start of the bins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1.0
+
+The bins of the grouping are adjusted based on the beginning of the day of the time series starting point. This works well with frequencies that are multiples of a day (like `30D`) or that divide a day evenly (like `90s` or `1min`). This can create inconsistencies with some frequencies that do not meet this criteria. To change this behavior you can specify a fixed Timestamp with the argument ``origin``.
+
+For example:
+
+.. ipython:: python
+
+    start, end = "2000-10-01 23:30:00", "2000-10-02 00:30:00"
+    middle = "2000-10-02 00:00:00"
+    rng = pd.date_range(start, end, freq="7min")
+    ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
+    ts
+
+Here we can see that, when not using ``origin``, the result after "2000-10-02 00:00:00" are not identical depending on the start of time series:
+
+.. ipython:: python
+
+    ts.resample("17min").sum()
```

Comment:
Can you build the docs to check this if you haven't already? I *think* only the second expression gets printed here, though we probably want both to highlight the difference

Reply:
I have build the doc before committing this, here is what I see:

![timeseries_doc_1](https://user-images.githubusercontent.com/3902859/78950658-bd3e0a00-7acf-11ea-91bb-a9d7118697bd.png)

Since we see the output of this line, I think I can set this conversation as Resolve safely :slightly_smiling_face: